i18n.js

```diff
@@ -330,6 +330,10 @@ i18n.prototype = {
 
 			locale = this.defaultLocale;
 
+			if ((this.devMode) && (fs.existsSync(this.locateFile(locale)))) {
```

Comment:
Why would this condition would match if the defaultLocale have parsing errors ?

Reply:
I think you want this to be over line 331

Reply:
This code matches (and throws an error) when json is present BUT is (for example) missing one comma...   You can test by creating a file with two comments and then remove the comma from the en.json file. Without this code the file is rewritten when a parse error occurs which potentially erases thousands of lines due to a missing comma :-)

This code prevents a json file from being overwritten when it has content but can not be parsed correctly.

Reply:
Don't think we want the extra line because the problem would occur in ANY language not just the default language...